pybind11_stubgen/__init__.py

```diff
@@ -221,6 +231,17 @@ def remove_signatures(docstring):  # type: (str) ->str
 
         return "\n".join(filter(lambda line: not re.match(signature_regex, line), lines))
 
+    @staticmethod
+    def filter_docstring(docstring):  # type: (str) ->str
+        docstring_new = StubsGenerator.remove_signatures(docstring)
+
+        # Reduce the double empty line to a single empty line
+        docstring_new = re.sub(r"\n\s*\n", "\n\n", docstring_new)
+        if docstring_new and re.match(r"^\s*$", docstring_new):
+            docstring_new = ""
```

Comment:
Those lines needs to be in another function, named something like `strip_repeated_empty_lines`.
Then implementation of `sanitize_docstring` will be imperative an concise:

```python
docstring = StubsGenerator.remove_signatures(docstring)
docstring = StubsGenerator.remove_strip_repeated_empty_lines(docstring)
return docstring
```